docs/modding/README.md

```diff
@@ -1,5 +1,6 @@
 # Modding
 
-The documentation for modding has been moved over to [r2northstar.readthedocs.io](https://r2northstar.readthedocs.io)
+### Modding documentation for Northstar and Titanfall 2 has moved over to: [https://r2northstar.readthedocs.io/](https://r2northstar.readthedocs.io/).
 
-For skin creation, see [https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/](https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/)
\ No newline at end of file
+# Custom Skin Modding
+### RETRY Titanfall 2 Custom Skin & Textures Wiki & Guides Gitbook is at: [https://retrywiki.gitbook.io/tf2/](https://retrywiki.gitbook.io/tf2/).
```

Comment:
Same as above, also smaller header cause not page title and headers need newlines after cause GitBook™. We should probably also add NS modding wiki here ^^

```suggestion
## Custom Skin Modding

RETRY Titanfall 2 Custom Skin & Textures Wiki & Guides Gitbook is at: [https://retrywiki.gitbook.io/tf2/](https://retrywiki.gitbook.io/tf2/).

## General Titanfall2 Modding

NoSkill Modding Guide: [https://noskill.gitbook.io/titanfall2/](https://noskill.gitbook.io/titanfall2/)
```

Reply:
Fixed. Done.